src/meta-srv/src/handler.rs

```diff
@@ -478,6 +478,9 @@ pub struct HeartbeatHandlerGroupBuilder {
     /// The handler to handle region lease.
     region_lease_handler: Option<RegionLeaseHandler>,
 
+    /// The heartbeat flush threshold.
+    heartbeat_flush_threshold: Option<usize>,
```

Comment:
I don't like this name. In factor, it's related to the heartbeat interval. So  I think we can call it `flush_stats_factor`, it looks better. It's not a threshold, but a factor.